### PR TITLE
Collate Media nodes

### PR DESCRIPTION
**Pull request by sjmiles**
The `container` workaround mentioned in last PR immediately bit us, lol. In particular when we have a Designer designing a Designer, the target container is no longer globally unique. 

Maybe `newNodeInfo` can contain some kind of target id (a Particle does not have access to it's own id, so it would probably be a property of Designer).

This is actually a good problem to encounter, as it supports us contending properly with the scoping issue.

### CODE REVIEW

File: pkg/Library/Layout/MaterialTabPagesNode.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+export const MaterialTabPagesNode = {
```
Comment (mariakleiner) [inline pkg/Library/Layout/MaterialTabPagesNode.js:6]: is this what i think this is? 🤯

**Review by mariakleiner — APPROVED** [review]
🥇

File: core/ts/recipe/Graphinator.ts [hunk 2]
```diff
@@ -170,9 +170,9 @@ export class Graphinator {
   findRemovedParticles(particles) {
     const runningGraphParticleIds = keys(this.arc.hosts).filter(id => {
       const container = this.arc.hosts[id].meta.container;
-      if (container === 'designer#graph') {
-        return true;
-      }
+      // if (container === 'designer#graph') {
```
[the remark this answers is not in the document]
Comment (sjmiles) [reply core/ts/recipe/Graphinator.ts:173]: Ok, thanks ... maybe we can test against `defaultContainer`.

File: pkg/Library/Layout/MaterialTabPagesNode.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/Layout/MaterialTabPagesNode.js:6]: I hope so. :)

**Pull request merged**